### ISSUE TITLE
Missing api_key error on Verify API (2FA)

### ISSUE THREAD

**Issue by romeoonisim** [Vonage/vonage-php-sdk-core#391]
On a blank PHP project, using the example code provided, I cannot make the Verify API work on the latest version, on 4.0, I get "Missing api_key" as an error. 

## Expected Behavior

I would expect the example code to work as expected out of the box

## Current Behavior
```
<br />
<b>Fatal error</b>:  Uncaught Vonage\Client\Exception\Request: Missing api_key in /Users/romeoonisim/Sites/test/vonagetest/vendor/vonage/client-core/src/Verify/ExceptionErrorHandler.php:56
Stack trace:
#0 /Users/romeoonisim/Sites/test/vonagetest/vendor/vonage/client-core/src/Client/APIResource.php(297): Vonage\Verify\ExceptionErrorHandler-&gt;__invoke(Object(GuzzleHttp\Psr7\Response), Object(Laminas\Diactoros\Request))
#1 /Users/romeoonisim/Sites/test/vonagetest/vendor/vonage/client-core/src/Client/APIResource.php(128): Vonage\Client\APIResource-&gt;getException(Object(GuzzleHttp\Psr7\Response), Object(Laminas\Diactoros\Request))
#2 /Users/romeoonisim/Sites/test/vonagetest/vendor/vonage/client-core/src/Verify/Client.php(89): Vonage\Client\APIResource-&gt;create(Array, '/json')
#3 /Users/romeoonisim/Sites/test/vonagetest/index.php(8): Vonage\Verify\Client-&gt;start(Object(Vonage\Verify\Verification))
#4 /Users/romeoonisim/.composer/vendor/laravel/valet/server.php(234): require('/Users/romeooni...')
#5 {main}
  thrown in <b>/Users/romeoonisim/Sites/test/vonagetest/vendor/vonage/client-core/src/Verify/ExceptionErrorHandler.php</b> on line <b>56</b><br />

```
## Possible Solution

It seems that the Verify/Client.php is not attaching the `api_key` and `api_secret` as body parameters, instead, it creates an Authorization header with the two values encoded in base64. Here are some of the documentation links:
https://developer.vonage.com/en/api/verify#verifyCheck
https://developer.vonage.com/en/getting-started/concepts/authentication 

## Steps to Reproduce (for bugs)

1. Create a new php project
2. Type in console `composer require vonage/client`
3. Create a index.php with the following code:
```
<?php
require_once 'vendor/autoload.php';

$client = new Vonage\Client(new Vonage\Client\Credentials\Basic(API_KEY, API_SECRET));

$request = new \Vonage\Verify\Request(PHONE_NUMBER, 'Testing');
$response = $client->verify()->start($request);

echo 'Started verification with an id of: '.$response->getRequestId();
```
4. Replace API_KEY, API_SECRET, PHONE_NUMBER
5. Execute code

## Context

This issue prevents me from upgrading to Laravel 9 on the project I'm working on

## Your Environment

* Version used: 4.0
* Environment name and version PHP 8.0.2 on nginx 1.23.3):
* Operating System and version:
System Version: macOS 12.1 (21C52) Apple M1
Kernel Version: Darwin 21.2.0

**Comment by SecondeJK** [comment]
This will be a fairly quick win, expect it to be rolled out with the release of Verify v2

**Comment by ngoctiennet963** [comment]
- Upgrading vonage/client-core (3.1.6 => 4.1.1): Extracting archive
After upgrade client-core 4 - I see the same issue when start verify api

Missing api_key
code 2

**Comment by SecondeJK** [comment]
Yep, incoming PR shortly.
It turns out the TokenBodyHandler for authentication this way never worked.

**Comment by romeoonisim** [comment]
@ngoctiennet963 if you are blocked, maybe my response from https://github.com/Vonage/vonage-laravel/issues/16
might help you until the PR will be available.

**Comment by SecondeJK** [comment]
Released under 4.1.2, thanks for reporting ✅

**Comment by kvenn** [comment]
I can't tell if I'm doing something wrong, but I can't find the version 4.1.2 in composer. Would appreciate any help, thank you!!

"vonage/client": "4.1.2"

  Problem 1
    - Root composer.json requires vonage/client 4.1.2 (exact version match: 4.1.2 or 4.1.2.0), found vonage/client[dev-feature/bump-for-v4, dev-main, 1.9.0, 1.9.1, 2.0.0, ..., 2.4.0, 3.0.0, 4.0.0] but it does not match the constraint.
